pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1007,18 +1001,48 @@ public void bootnodesOptionMustBeUsed() {
 
   @Test
   public void bannedNodeIdsOptionMustBeUsed() {
-    final String[] nodes = {"0001", "0002", "0003"};
-    parseCommand("--banned-node-ids", String.join(",", nodes));
-
-    verify(mockRunnerBuilder).bannedNodeIds(stringListArgumentCaptor.capture());
+    final BytesValue[] nodes = {
+      BytesValue.fromHexString(
+          "6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0"),
+      BytesValue.fromHexString(
+          "7f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0"),
+      BytesValue.fromHexString(
+          "0x8f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0")
+    };
+
+    final String nodeIdsArg =
+        Arrays.asList(nodes).stream()
+            .map(BytesValue::toUnprefixedString)
+            .collect(Collectors.joining(","));
+    parseCommand("--banned-node-ids", nodeIdsArg);
+
+    verify(mockRunnerBuilder).bannedNodeIds(bytesValueCollectionCollector.capture());
     verify(mockRunnerBuilder).build();
 
-    assertThat(stringListArgumentCaptor.getValue().toArray()).isEqualTo(nodes);
+    assertThat(bytesValueCollectionCollector.getValue().toArray()).isEqualTo(nodes);
 
     assertThat(commandOutput.toString()).isEmpty();
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+  @Test
+  public void callingWithBannedNodeidsOptionButNoValueMustDisplayErrorAndUsage() {
```

Comment:
Can these be moved to 986?  It makes the diff more comprehensible.

Reply:
The bannedNodeIds test was sandwiched in between some bootnode tests, so moved things around a bit for a more coherent test layout 🤷‍♀